File: java/TuduManager/web.api.spring.boot/src/main/java/net/seymourpoler/tudumanager/web/api/spring/boot/todo/DeleteTodoController.java
package net.seymourpoler.tudumanager.web.api.spring.boot.todo;

import net.seymourpoler.tudumanager.domain.todo.delete.IDeleteTodoService;
import org.springframework.beans.factory.annotation.Autowired;
import org.springframework.http.HttpStatus;
import org.springframework.http.ResponseEntity;
import org.springframework.web.bind.annotation.DeleteMapping;
import org.springframework.web.bind.annotation.RequestParam;
import org.springframework.web.bind.annotation.RestController;

@RestController
public class DeleteTodoController {
    private final IDeleteTodoService deleteTodoService;

    @Autowired
    public DeleteTodoController(IDeleteTodoService deleteTodoService) {
        this.deleteTodoService = deleteTodoService;
    }

    @DeleteMapping("/api/todos")
    public ResponseEntity delete(@RequestParam("todoId") Integer todoId){
        var executionResult = deleteTodoService.delete(todoId);
        if(executionResult.isOk()) {
            return new ResponseEntity(HttpStatus.OK);
        }
        return new ResponseEntity(HttpStatus.NOT_FOUND);
    }
}

File: dotnet/react.mvc-core.postgres.dapper/TuduManager/src/TuduManayer.web.api/Todo/Delete/DeleteTodoController.cs
using Microsoft.AspNetCore.Mvc;
using TuduManayer.Domain.Todo.Delete;

namespace TuduManayer.web.api.Todo.Delete
{
    public class DeleteTodoController : Controller
    {
        private readonly IDeleteTodoService service;

        public DeleteTodoController(IDeleteTodoService service)
        {
            this.service = service;
        }

        [HttpDelete("/api/todos/{todoId}")]
        public IActionResult Delete(int todoId)
        {
            var resultExecution = service.Delete(todoId);
            if (resultExecution.IsOk)
            {
                return Ok();
            }

            return NotFound();
        }
    }
}